actions/checkForAndPlaceOrder.ts

```diff
@@ -85,6 +97,11 @@ const _checkForAndPlaceOrder: ActionFn = async (
     blockNumber
   );
 
+  console.log(
+    "[checkForAndPlaceOrder] Number of orders: ",
+    Array.from(registry.ownerOrders.values()).flatMap((s) => s).length
+  );
```

Comment:
Log the number of orders. Happy to remove it, for now its helpful to see clearly the number of orders at the beginning and end of the execution